test/pkg/environment/common/environment.go

```diff
@@ -157,10 +159,11 @@ func (env *Environment) DefaultNodeClass() *v1alpha1.KWOKNodeClass {
 
 func (env *Environment) DefaultNodePool(nodeClass *v1alpha1.KWOKNodeClass) *v1beta1.NodePool {
 	nodePool := test.NodePool()
+	gvk := object.GVK(nodeClass)
 	nodePool.Spec.Template.Spec.NodeClassRef = &v1beta1.NodeClassReference{
 		Name:       "default",
-		Kind:       "KWOKNodeClass",
-		APIVersion: v1alpha1.SchemeGroupVersion.Version,
+		Kind:       gvk.Kind,
```

Comment:
```suggestion
		Kind:       object.GVK(nodeClass).Kind,
```

consider inlining this -- I don't think it's expensive to do so